modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3725,9 +3737,11 @@ public CodegenOperation fromOperation(String path,
         String operationId = getOrGenerateOperationId(operation, path, httpMethod);
         // remove prefix in operationId
         if (removeOperationIdPrefix) {
-            int offset = operationId.indexOf('_');
-            if (offset > -1) {
-                operationId = operationId.substring(offset + 1);
+            String[] componenets = operationId.split("[" + removeOperationIdPrefixDelimiter + "]");
```

Comment:
Suggestion: adding a 1-liner comment explaining what the code block does

Reply:
Done

Reply:
👍